docs/content/develop/properties.md

```diff
@@ -0,0 +1,352 @@
+
+Example, cascading an attribute `pfe-c-foo` to any `h1`, `h2`, or `h3` child elements.
+
+**<pfe-foo>**
+```javascript
+static get properties() {
+  return {
+    foo: {
+      type: String,
+      cascade: "h1,h2,h3"
+    }
+  };
+}
+```
+
+With this property definition, any `h1`, `h2`, or `h3` children will get the same attribute and value, such as:
+
+**Before connectedCallback**
+```html
+<pfe-foo foo="hello">
+    <h1></h1>
+</pfe-foo>
+```
+
+**After connectedCallback**
+```html
+<pfe-foo pfe-c-foo="hello">
+    <h1 pfe-c-foo="hello"></h1>
+</pfe-foo>
+```
+
+Attribute values are only copied from parent to child, never the other way around.
+
+## prefix
+
+PatternFly Elements has a convention of prefixing a component's own attribute names with `pfe-c-`, and the `prefix` field controls that.  Attributes are prefixed by default, but you may set `prefix: false` if you wish your attribute to have no prefix.
+
+This can be useful when deprecating old, non-prefixed attributes.  It's also beneficial when using built-in aria attributes to manage component states, such as `aria-hidden`:
+
+```javascript
+static get properties() {
+  return {
+    ariaHidden: {
+      type: String,
+      default: "true",
+      prefix: false
+    }
+  };
+}
+```
+
+The property name `ariaHidden` results in an attribute named `aria-hidden`.  If prefix were not set to false, the generated attribute name would be `pfe-c-aria-hidden` instead.
+
+Note: [ARIA attributes](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) must be defined as Strings, for even though attributes like `aria-hidden` seem like Booleans, they are not true boolean attributes (`aria-hidden` vs. `aria-hidden="true"`) and need to receive string values of `"true"` and `"false"`.
+
+## alias
+
+Default: `undefined`
+
+The `alias` field can be used to link two properties to each other.  `alias` expects to be given the name of another property, to which it will copy its value.  When a property value is changed, if it has an alias, the new value will be sent to the alias.
+
+Aliasing is intended to help when migrating to a new property/attribute name.  It allows for two different properties to be simultaneously "active".
+
+Aliased values do not need to have a new set of definitions for cascade or observer as those states will be managed by the attribute to which it is being aliased (see example below).
+
+Example, an `old` property whose values will be forwarded along to a `new` property.
+
+```javascript
+static get properties() {
+  return {
+    new: {
+      type: Number,
+      default: 0,
+    },
+    old: {
+      type: Number,
+      alias: "new"
+    }
+  };
+}
+```
+
+With these property definitions, changes to `old` will be "forwarded" along to `new`.
+
+```javascript
+el.source = 4;
+el.destination === 4; // true
+```
+
+Two-way aliases are allowed.
```

Comment:
I'm unsure of why you would use a two-way alias?

Reply:
What's important is that two-way aliasing doesn't fail catastrophically.  It would be very easy for it to become an infinite loop.  "Support" for two-way aliasing means not falling over dead when it's done.

But for the sake of argument, two-way aliasing could be used to keep two attributes in sync with each other, and that could be useful.  For example, let's say we're integrating a component with some third-party library or framework, and that integration required a framework-specific attribute.  We should not lock in to that attribute, so we create our own custom attribute that aliases to the framework-specific attribute, _and back_.  That way, we can keep on supporting our own custom attribute even if we remove that integration or add another integration.